docs/users-guide/expressions.md

```diff
@@ -0,0 +1,102 @@
+| Starts with        | `startsWith(text, comparison)`                 | Returns true if the beginning of the text matches the comparison text.                                                                                             | `startsWith( [Course Name], "Computer Science" )`                    |
+| Standard deviation | `StandardDeviation(column)`                    | Calculates the standard deviation of the column.                                                                                                                   | `StandardDeviation( [Population] )`                                  |
+| Substring          | `substring(text, position, length)`            | Returns a portion of the supplied text.                                                                                                                            | `substring( [Title], 0, 10 )`                                        |
+| Sum                | `sum(column)`                                  | Adds up all the values of the column.                                                                                                                              | `sum( [Subtotal]` )                                                  |
+| Sum if             | `sumif(column, condition)`                     | Sums up the specified column only for rows where the condition is true.                                                                                            | `sumif( [Subtotal], [Order Status] = "Valid" )`                      |
+| Square root        | `sqrt(number)`                                 | Returns the square root.                                                                                                                                           | `sqrt([Hypotenuse])`                                                 |
+| Trim               | `trim(string)`                                 | Removes leading and trailing whitespace from a string of text.                                                                                                     | `trim( [Comment] )`                                                  |
+| Upper              | `upper(text)`                                  | Returns the text in all upper case.                                                                                                                                | `upper( [Status] )`                                                  |
+| Variance           | `variance(column)`                             | Returns the numeric variance for a given column.                                                                                                                   | `variance([Temperature])`                                            |
+
+### Database limitations
+
+Certain database types don't support some of the above functions:
```

Comment:
1. Is this an exhaustive list? 
2. How does this impact 3rd-party drivers? I know we're planning to update [the wiki](https://github.com/metabase/metabase/wiki/What's-new-in-0.35.0-for-Metabase-driver-authors) so we may want to comment here and link out to that.

Reply:
1. This is going off of the spreadsheet that @sbelak provided, so it's exhaustive as far as I know.
2. Cool, good idea. I'll add a link to the wiki.

Reply:
Checked out the spreadsheet and, yes, it's exhaustive. 🙌 We should def make that public as it's a great reference.